src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -131,21 +131,6 @@ impl LevelCompactionPicker {
         level_handlers: &[LevelHandler],
         stats: &mut LocalPickerStatistic,
     ) -> Option<CompactionInput> {
-        // TODO: remove this
-        let l0_size = l0.total_file_size - level_handlers[0].get_pending_file_size();
```

Comment:
I think this condition shall keep until https://github.com/risingwavelabs/risingwave/pull/11903/ has been merged.

Reply:
Due to test , i remove this condition.

Avoid having all of our compactions rejected by this condition, which can happen when lowering the write limit. I tested the anti-aff deployment with no significant performance difference.